src/cluster_asm.c

```diff
@@ -921,7 +919,7 @@ static void replyTaskStatus(client *c, asmTask *task) {
     addReplyBulkCString(c, "id");
     addReplyBulkCString(c, task->id);
     addReplyBulkCString(c, "slots_range");
```

Comment:
`slots_range` to `slots`?

Reply:
hmm, I thought you did in your PR. I'll add it